Makefile

```diff
@@ -87,17 +87,17 @@ docs/%.5: docs/%.5.md
 
 docs: $(MANPAGES_MD:%.md=%)
 
-install: ocid.conf
+install:
```

Comment:
to install? I don't like that, you usually do "make && sudo make install" - re-compiling on make install?